test/e2e/libpod_suite_test.go

```diff
@@ -13,7 +13,6 @@ import (
 )
 
 func SkipIfRemote() {
-	ginkgo.Skip("This function is not enabled for remote podman")
```

Comment:
Why are you deleting this line?

Reply:
because skipifremote should not do anything when testing "locally"